src/modules/dashboard/components/ManageFundsDialog/ManageFundsDialog.tsx

```diff
@@ -118,9 +118,7 @@ const ManageFundsDialog = ({
     : colony.canMintNativeToken && hasRoot(allUserRoles);
   const canUnlockToken = isVotingExtensionEnabled
     ? colony.isNativeTokenLocked && colony.canUnlockNativeToken
```

Comment:
You also need to change this condition, as of now I can't access the dialog even if I have root permission when the voting reputation extension is enabled:
![FireShot Capture 504 - Colony - localhost](https://user-images.githubusercontent.com/18473896/144507162-7643234f-9882-4793-bdbd-ec2ca04ee9e5.png)

Reply:
I've fixed it by removing `colony.isNativeTokenLocked &&` in line 120.

Tested by enabling motions and working as expected, thanks for catching this as I didn't think of it